nemo/collections/common/tokenizers/huggingface/auto_tokenizer.py

```diff
@@ -106,28 +106,24 @@ def __init__(
         else:
             self.tokenizer = AUTOTOKENIZER.from_pretrained(pretrained_model_name_or_path=pretrained_model_name)
 
-        self.tokenizer.eos_token = self.tokenizer.sep_token
-        self.tokenizer.bos_token = self.tokenizer.cls_token
-
         special_tokens_dict = {}
-        if unk_token:
+        if self.tokenizer.unk_token is None:
```

Comment:
"is None" or "is not None"?

Reply:
yes, we're making sure that the default tokenizer's special tokens are preserved